src/views/Home.vue

```diff
@@ -1,16 +1,18 @@
 <template>
   <maps v-if="$route.params.id === 'maps'" />
   <charts v-else-if="$route.params.id === 'charts'" />
+  <chat v-else-if="$route.params.id === 'chat'" />
   <project v-else />
 </template>
 
 <script>
   import Project from "@/components/Project"
   import Charts from "@/components/Charts"
   import Maps from "@/components/Maps"
+  import Chat from "@/components/Chat/ChatComponent.vue"
```

Comment:
can we rename Chat/ChatCompoent.vue to Chat/index.vue and use it like folder above?

Reply:
Rename ChatCompoent.vue to index.vue